docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  ove-apps-latest:
+    image: ove/ove-apps:latest
+    build: .
+    ports:
+    - "8081:8081"
+    - "8082:8082"
+    - "8083:8083"
+    - "8084:8084"
+    - "8085:8085"
+    - "8086:8086"
+    - "8087:8087"
+    - "8088:8088"
+    - "8089:8089"
+    environment:
+      # this is not going to work because localhost is reserved for the local docker network
```

Comment:
This is not an issue right? The hostnames are resolved in-VM and also in most cases on the browser, so even right now on the dev environment we don't use localhost but the actual hostname allocated to it.

However, host-ip would not work, since that may not resolve internally. How we generally get things like this to work is to have a locally resolving IP associated with the hostname and an externally resolving IP associated with the same hostname. So, the line below would need an edit.

Reply:
Docker images run on their own network by default (different network adapter, etc) to isolate traffic. Therefore, localhost will point to the local ip of the container and not the hostname ip. The comment is right, if you need to make this work you have to put in the hostname ip or the docker name of the OVE service, which will resolve into the docker ip if the two containers are part of the same network.

Reply:
@oserban, as discussed the point vs WRT calling this `hostname` instead of `hostname ip`.